chicken-sloths/bangazon-site-sprint3
# Manage Products View/'Delete' Product

# Description
User can view the products they have added and then "delete" them - which is actually a patch to change deleted: false to deleted: true. It then removes it from the view and also from future views due to conditional in product-card template only displaying products with deleted: false. This also handles different product card views where only those with the state of "manage" have the capability to delete products.

## Related Ticket(s)
#11  #39 

## Problem to Solve
Ability to view all user prods and remove them from sale

## Proposed Changes
file names if you want - I am not particular and just named the client side js file in a literal context. 

## Expected Behavior
user can view their products and delete them

## Steps to Test Solution

1. ```npm start```
2. go to home page of bangazon and log in
3. go to "manage products" and you should only see your products (if you have any...)
4. click "add new product" and you will be taken to the add to product form. fill it out and submit (this resolves with json and will need to be updated with another ticket, I added the ticket to our project board #60)
5. go back to manage products, you will see that product displayed.
6. click the hyperlink to see that products details
7. go back to manage products, click delete next to a product, it will be removed from the DOM, if you check in pgadmin or squelectron that product's deleted boolean should now be true! 
8. go out of manage products or reload, you will see that the deleted product is no longer in view.
